hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+                   `sck`, `cs_n` and `sd` signals are no longer driven by
+                   the SPI_HOST IP, rather they are multiplexed out to match the
+                   on-chip inputs `pt_sck_i`, `pt_cs_ni`, `pt_sdt_i`, `pt_sdo_i`,
+                   and `pt_sdi_o`.  (Since the `sd` bus is bidirectional, there are
+                   separate input, output and tri-state controls for this bus.)''',
+          resval: "0x0"
+        },
+        { bits: "24:16",
+          name: "TX_WATERMARK"
+          desc: '''The IP can optionally send an interrupt if the
+                   depth of the TX FIFO is ever less than
+                   TX_WATERMARK words (32b each). Set to zero
+                   to temporarily disable interrupts.'''
+          resval: "0"
+        },
+        { bits: "15:9",
+          name: "RX_WATERMARK"
+          desc: '''The IP can optionally send an interrupt if the
+                   depth of the RX FIFO is ever greater than
+                   RX_WATERMARK words (32b each).'''
+          resval: "127"
+        },
+        { bits: "8",
+          name: "MANCS_EN",
+          desc: '''Enable Manual Chip Select. Typically each command
+                   manages the CS lines automatically, issuing a command
+                   to exactly one device, and lowering and raising the
+                   appropriate CS line after transmitting or receiving the
+                   necessary bytes.  Asserting this bit overrides this
+                   automatic selection, instead setting the CS lines to
+                   reflect the contents of the MANUAL_CS field.'''
+          resval: "0x0"
+        },
+        { bits: "7:0",
+          name: "MANUAL_CS"
+          desc: '''When MANCS_EN = 1, the cs_n lines are directly to the
+                   value of MANUAL_CS.'''
+          resval: 0
+        }
+      ]
+    },
+    { name: "STATUS",
+      desc: "Status register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "31",
+          name: "READY",
+          desc: '''When high, indicates the SPI host is ready to receive
+                   commands. Writing to SPICMD when RDY is low is an error,
+                   and will trigger an interrupt.
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "30",
+          name: "ACTIVE",
+          desc: '''When high, indicates the SPI host is processing a previously
+                   issued command.'''
+          resval: "0x0"
+        },
+        { bits: "29",
+          name: "TXFULL",
+          desc: '''When high, indicates that the transmit data fifo is full.
+                   Any further writes to TXD will create an error interrupt.
+                '''
+          resval: "0x0"
+        },
+        { bits: "28",
+          name: "TXEMPTY",
+          desc: '''When high, indicates that the transmit data fifo is empty.
+                   And pending commands may have to wait for more data.
```

Comment:
Should this be "any" pending commands? Also, I'm not sure about the "may": surely, they must?

Reply:
It's true, any commands (if the existed) would be stuck at this point. However to dig into the grammar weeds a bit, if there is some uncertainty as to whether a command exists, don't you have express this in the subjunctive... somehow?  Maybe not in English... nobody cares about the subjunctive mood in English I suppose.

Though the cleanest way I see of fixing this is to delete the whole sentence.